docs/spring-boot-runtime/master.adoc

```diff
@@ -14,6 +14,92 @@ include::topics/sb-tomcat-runtime-overview.adoc[leveloffset=+1]
 
 {SpringBoot} has a version of its runtime that is tested and verified to run on OpenShift. Some components, for example, the Embedded Tomcat Container and the Java container image, are provided as part of a Red Hat subscription when used with {SpringBoot}.
 
+The features in the table below are available as a collection of missions which run on OpenShift. Some features are native as part of Kubernetes, others are available from link:https://github.com/spring-cloud-incubator/spring-cloud-kubernetes[Spring Cloud].
+
+.Features and Frameworks Summary
+[width="100%",options="header"]
+|====================================================================
+|Feature |Problem Addressed |Cloud Native |Framework
+
+a|
+Circuit Breaker
+
+a|
+Switch between services and continue to process incoming requests without interruption in case of service failure.
+
+a|
+Yes - using Kubernetes API
+
+a|
+Spring Cloud Kubernetes - Hystrix
+
+
+a|
+Health Check
+
+a|
+Check readiness and liveness of the service based on information exposed by Spring Boot Actuator. Service restarts automatically if probing fails.
```

Comment:
I think liveness and readiness are also addressed in Kubernetes (hence the yes in native column) so maybe we shouldn't mention Actuator here?